smartsim/entity/application.py

```diff
@@ -242,27 +245,6 @@ def attached_files_table(self) -> str:
             return "No file attached to this application."
         return str(self.files)
 
-    def print_attached_files(self) -> None:
-        """Print a table of the attached files on std out"""
-        print(self.attached_files_table)
-
-    def __str__(self) -> str:  # pragma: no cover
-        exe_args_str = "\n".join(self.exe_args)
-        entities_str = "\n".join(str(entity) for entity in self.incoming_entities)
-        return textwrap.dedent(f"""\
-            Name: {self.name}
-            Type: {self.type}
-            Executable:
-            {self.exe}
-            Executable Arguments:
-            {exe_args_str}
-            Entity Files: {self.files}
-            File Parameters: {self.file_parameters}
-            Incoming Entities:
-            {entities_str}
-            Key Prefixing Enabled: {self.key_prefixing_enabled}
-            """)
```

Comment:
Not wrong just curious: is there a reason that these methods were moved? Is it just an artifact of nasty merge conflict? If so, feel free to just resolve this comment!

Reply:
A similar question also applies to `add_exe_args`

Reply:
they are all still in the file! They are just moved